storybookjs/storybook
# Build lib/addons with ts-up

Issue: #18732

## What I did
Changed addons in lib to be compiled by the bundler ( ts-up )
## How to test

- [ YES ] Is this testable with Jest or Chromatic screenshots? Testable by the original tests
- [ NO ] Does this need a new example in the kitchen sink apps?
- [ NO ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
